packages/app/src/components/SearchResultsCard/index.js

```diff
@@ -1,42 +1,106 @@
-import React, { useContext, useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
-import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
-import './style.css';
+import React, { useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { SearchResultDataContext } from "../../utils/context/SearchResultsContext";
+import "./style.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const SearchResultsCard = () => {
-  const [SearchResultData, setSearchResultData] = useContext(SearchResultDataContext);
+  const [SearchResultData, setSearchResultData] = useContext(
+    SearchResultDataContext
+  );
+
+  SearchResultData.sort((a, b) => {
+    let patient1 = a.patientName;
+    let patient2 = b.patientName;
+
+    if (patient1 > patient2) {
+      return 1;
+    }
+
+    if (patient2 > patient1) {
+      return -1;
+    }
+    return 0;
+  });
+
   return (
     <>
       <section className="search-result-card">
-        <table className="search-result-table">
+        <table className="search-result-table search-result-data">
           <thead>
             <tr>
-              {SearchResultData.length > 0 ?<th> Search Results</th>: <th>No Applications Found</th>}
+              {SearchResultData.length > 0 ? (
```

Comment:
Instead of doing `SearchResultData.length > 0 ? stuffToRender : null` you can do `SearchResultData && stuffToRender` which will effectively function the same way.